Swat/SwatHtmlTag.php

```diff
@@ -313,10 +274,10 @@ private function openInternal($self_closing = false)
         foreach ($this->attributes as $attribute => $value) {
             if ($value !== null) {
                 echo ' ',
-                    $attribute,
-                    '="',
-                    SwatString::minimizeEntities($value),
-                    '"';
+                $attribute,
```

Comment:
weird echo formatting

Reply:
Again, because of `echo` with commas vs concatenation.